apps/base-docs/docs/connecting-to-the-blockchain/overview-of-providers.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Overview of Providers
+description: Learn what providers are and why you need one.
+hide_table_of_contents: false
+---
+
+Onchain apps need frontends, sometimes called dApps, to enable your users to interact with your smart contracts. A _provider_ makes the connection from frontend to blockchain, and is used to read data and send transactions.
+
+---
+
+## Prerequisites
+
+Before this lesson, you should:
+
+- Be familiar with modern, frontend web development
+- Possess a general understanding of the EVM and smart contracts
```

Comment:
This bullet point provides an opportunity to link/point readers to Base Camp content to gain an understanding if they don't have one.